SECURITY.md

```diff
@@ -0,0 +1,5 @@
+# Security Policy
+
+## Reporting a Vulnerability
+
+Please report security vulnerabilities to xxxxxxxxx@xxxxxxxxxxxxxx.xxx 
```

Comment:
pangeo-steering-council@googlegroups.com ???

Not sure what to do here actually.

Reply:
```suggestion
Please report security vulnerabilities to pangeo-steering-council@googlegroups.com 
```